docker/readme.md

```diff
@@ -15,13 +15,20 @@ The following tags are available:
 
 - `latest` / `ubuntu-latest` - contains the latest version of RavenDB running on Ubuntu container
 
-- `windows-latest` - contains the latest version of RavenDB running running on Windows nanoserver
+- `windows-1809-latest` - contains the latest version of RavenDB running running on Windows nanoserver (kernel 1809)
+
+- `windows-ltsc2022-latest` - contains the latest version of RavenDB running running on Windows nanoserver (kernel 2022)
```

Comment:
the thing in parentheses - are you sure this is kernel version? 
Wikipedia says it's `version`